step-framework-parent/step-framework-timeseries/src/main/java/step/core/timeseries/ingestion/TimeSeriesIngestionChain.java

```diff
@@ -0,0 +1,24 @@
+package step.core.timeseries.ingestion;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class TimeSeriesIngestionChain {
```

Comment:
These classes doesn't seem to be used and to be redundant with the new implementation in TimeSeriesBuilder, what's the goal?

Reply:
it is an old approach. will be removed completely.